scripts/tcl_commands/synthesis.tcl

```diff
@@ -243,29 +243,68 @@ proc logic_equiv_check {args} {
     exec echo "[TIMER::get_runtime]" | python3 $::env(SCRIPTS_DIR)/write_runtime.py "logic equivalence check - yosys"
 }
 
+
+proc generate_blackbox_verilog {inputs output {defines ""}} {
+    set output_files ""
+    set defines_flag ""
+    set ::env(YOSYS_IN) $inputs
+    set ::env(YOSYS_OUT) $output
+    if { $defines != "" } {
+        set ::env(YOSYS_DEFINES) $defines
+    }
+    try_exec yosys -c $::env(SCRIPTS_DIR)/yosys/blackbox.tcl
+    puts_info "Generated blackbox verilog ($output) from ($inputs)"
+}
+
+
 proc run_verilator {} {
-    set verilator_verified_pdks "sky130A sky130B"
-    set verilator_verified_scl "sky130_fd_sc_hd"
-    set includes ""
-    if { [string match *$::env(PDK)* $verilator_verified_pdks] == 0 || \
-        [string match *$::env(STD_CELL_LIBRARY)* $verilator_verified_scl] == 0} {
-        puts_warn "PDK '$::env(PDK)', SCL '$::env(STD_CELL_LIBRARY)' will generate errors with instantiated stdcells in the design."
-        puts_warn "Either disable QUIT_ON_LINTER_ERRORS or remove the instantiated cells."
-    } else {
-        set pdk_verilog_models [glob $::env(PDK_ROOT)/$::env(PDK)/libs.ref/$::env(STD_CELL_LIBRARY_OPT)/verilog/*.v]
-        foreach model $pdk_verilog_models {
-            set includes "$includes $model"
-        }
+    set pdk_model_blackbox ""
+    if { $::env(PDK) == "sky130A" ||$::env(PDK) == "sky130B" } {
```

Comment:
I think the model generation should be for all PDKs and SCLs, not gated by this statement here. WHEN a problem arises, we can look into it.